### PR TITLE
[RW-2322][RW-2285][risk=no] Add list search and hierarchy view for Conditions 

### PR DESCRIPTION
**Pull request by dolbeew**
Add option for Conditions to the new criteria dropdown

*Changes are behind flags `enableListSearch` for api  and `enableCBListSearch` for ui

### CODE REVIEW

File: api/config/config_local.json [hunk 1]
```diff
@@ -60,6 +60,6 @@
     "enableBetaAccess": true
   },
   "cohortbuilder": {
-    "enableListSearch": false
+    "enableListSearch": true
```
Comment (freemabd) [inline api/config/config_local.json]: may want to keep this set to false.

File: ui/src/app/cohort-search/list-search/list-search.component.tsx [hunk 2]
```diff
@@ -140,28 +140,28 @@ export const ListSearch = withCurrentWorkspace()(
...
+      //   isStandard: null,
+      //   name: 'Aspartate aminotransferase',
+      //   parentId: 379173,
+      //   path: '379101.379103.379133.379173',
+      //   selectable: false,
+      //   subtype: 'LAB',
+      //   type: 'MEAS',
+      //   value: null
+      // };
+      // this.props.hierarchy(tempRow);
```
Comment (freemabd) [inline ui/src/app/cohort-search/list-search/list-search.component.tsx]: If this is not needed we should remove it?

File: api/config/config_local.json (hunk 1 above)
Comment (dolbeew) [reply api/config/config_local.json]: Done

File: ui/src/app/cohort-search/list-search/list-search.component.tsx (hunk 2 above)
Comment (dolbeew) [reply ui/src/app/cohort-search/list-search/list-search.component.tsx]: Done

**Pull request merged**